scripts/update-charts-dep.sh

```diff
@@ -28,11 +30,15 @@ declare -a charts=(
         mojaloop/reporting-hub-bop-trx-ui
         mojaloop/reporting-legacy-api
         mojaloop/security-hub-bop-kratos-ui
+        #mojaloop/security-role-perm-crd
```

Comment:
This does not exist anymore, so I would recommend removing it.

Reply:
will do

Reply:
done !